src/main/java/com/flightstats/hub/spoke/FileSpokeStore.java

```diff
@@ -178,14 +181,19 @@ private void keysInBucket(String key, OutputStream output) {
         }
         try {
             Collection<File> files;
-            if (resolution.equals("second")) {
+            if (Optional.ofNullable(resolution).orElse("FALSE").equals("second")) {
```

Comment:
I'm probably too oldschool but is this preferred style? or does `"second".equals(resolution)` solve the NPE? (This is more a question than a request to change this)

Reply:
I was going to say the same thing. I'm not seeing value in the Optional here.

Reply:
done